client/src/components/pages/ProgramInformation/index.js

```diff
@@ -8,13 +9,33 @@ class ProgramInformation extends Component {
   constructor(props) {
     super(props);
   }
+  state = {
+    result: []
+  };
+
+  componentDidMount() {
+    const { id } = this.props.match.params;
+    this.getActivitesByProgramId(id);
+  }
+
+  getActivitesByProgramId = programId => {
+    axios
+      .get(`/activities/getActivitesByProgramId/${programId}`)
+      .then(response => {
+        const result = response.data.activities;
+        this.setState({ result: result });
```

Comment:
You don't need `result: result` here as they called the same thing. You can just write `this.setState({ result });`